### ISSUE TITLE
Client side playlists.

### ISSUE THREAD

**Issue by mukkachaitanya** [OSDLabs/Encore#5]
Current app only allows the admin to make playlists and these playlists sync on the client side too. One would also want to allow the client to make his own client playlists, which can be stored locally on the client side.

**Comment by SebastinSanty** [comment]
I guess it allows clients to make playlists too!

**Comment by 0xRampey** [comment]
Yes it does. However the changes seem to reflect for everyone. That's pretty weird.

**Comment by mukkachaitanya** [comment]
No no it doesn't allow us to make client side playlists! It allows the clients to make playlist which is stored in the main database.

**Comment by 0xRampey** [comment]
Hmm.. okay. So we need to setup a lightweight database client-side right?

**Comment by SebastinSanty** [comment]
@mukkachaitanya I think that is what @prampey meant by saying that. We can use [Preferences](https://www.npmjs.com/package/preferences) instead of using a db, because, given db's location, there is a possibility, someone can fiddle/corrupt it. My two cents.

**Comment by 0xRampey** [comment]
@SebastinSanty That's a node module. We can't use that client-side. From what I see, there seem to be 2 ways to go:
- Save all custom playlist metadata on the server, for each user; serve it when required.
- Use WebStorage or WebSQL to store it locally.

**Comment by 0xRampey** [comment]
First one might require a lot of code changes and we should be able to identify users properly. 
Second one, users might be able to delete metadata after clearing cache.

**Comment by SebastinSanty** [comment]
@mukkachaitanya Are you still doing this? I might want to give a shot at this! Thanks :)

**Comment by mukkachaitanya** [comment]
@SebastinSanty Try it out. Do tell if you need any help :D
